### PR TITLE
Fixed Queue.get and Queue.put return types.

### PR DESCRIPTION
**Pull request by ze-phyr-us**
These methods need to return `Future[T]` and `Future`, respectively. More details in https://github.com/JukkaL/mypy/issues/836

### CODE REVIEW

File: stubs/3.4/asyncio/queues.pyi [hunk 1]
```diff
@@ -28,10 +29,10 @@ class Queue(Generic[T]):
     def empty(self) -> bool: ...
     def full(self) -> bool: ...
     @coroutine
-    def put(self, item: T) -> None: ...
+    def put(self, item: T) -> Future: ...
```
Comment (JukkaL) [inline stubs/3.4/asyncio/queues.pyi]: Could the return type be more specific? If not, probably better to use `Future[Any]`.

Comment (rockneurotiko) [inline stubs/3.4/asyncio/queues.pyi]: Probably are `Future[None]` because put don't return nothing (I think)

**Comment by ze-phyr-us** [comment]
Right. I've rebased on the current master and changed the return type to `Future[None]`.

**Comment by JukkaL** [comment]
Thanks, looks good now!

**Comment by matthiaskramm** [comment]
Applied this change to http://github.com/python/typeshed, as well.

**Pull request merged**